specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json

```diff
@@ -603,7 +603,7 @@
             "$ref": "./examples/SearchServiceCreateOrUpdateSkillset.json"
           }
         },
-        "description": "Creates a new cognitive skillset in an Azure Search service or updates the skillset if it already exists.",
+        "description": "Creates a new cognitive skillset in a search service or updates the skillset if it already exists.",
```

Comment:
same comment as in the other file...
"cognitive skillset/skillsets/skills" -> skillset | skillsets | skills